tests/rptest/tests/franz_go_verifiable_test.py

```diff
@@ -148,18 +144,27 @@ def __init__(self, ctx):
             si_settings=si_settings)
 
     @cluster(num_nodes=4)
-    def test_with_all_type_of_loads_and_si(self):
+    @matrix(timeboxed=[True, False], segment_size=[100 * 2**20, 20 * 2**20])
```

Comment:
Aim for an order of magnitude difference between params: e.g. 1MB and 16MB might be good choices here

Reply:
Sure I'll change the values but isn't 100MB vs 20MB an order of magnitude difference regardless?

Reply:
hah, I think noah and I both misread this the same way as being 10mb/20mb instead of 20mb/100mb.  Never mind!

Reply:
indeed I did misread it. it's some kind of illusion